### ISSUE TITLE
Support requirements in docstring

### ISSUE THREAD

**Issue by ralsina** [PyAr/fades#31]
It would be nice if, in addition to comments on imports, you could support a requirements.txt-like thing in the script's docstring, like this:

````python
#!/usr/bin/python

"""
This script to foobar the bazbats has the following requirements:

foo >=1.0
bar <2
bat
baz
"""
````

Syntax, parsing, etc are left as exercise for the authors ;-)

**Comment by facundobatista** [comment]
I would include a "fades flag", that would be a line with the fades word, and then indentation, like...

"""Script superpoderoso.

Requirements for fades:
    foo >= 1.0
    bar
"""

**Comment by facundobatista** [comment]
I'd postpone this from Release 3...

**Comment by gilgamezh** [comment]
>I'd postpone this from Release 3...

:+1:

**Comment by facundobatista** [comment]
This was just closed by #111.
